build/templates/class.rst.mako

```diff
@@ -62,6 +62,61 @@ ${helper.get_rst_header_snippet(item, '-')}
 
 % endfor
 
+% if len(config['repeated_capabilities']) > 0:
+.. role:: c(code)
+    :language: c
+
+.. role:: python(code)
+    :language: python
+
+${helper.get_rst_header_snippet('Repeated Capabilities', '=')}
+
+    Repeated capabilities attributes are used to set the `channel_string` parameter to the
+    underlying driver function call. This can be the actual function based on the :py:class:`Session`
+    method being called, or it can be the appropriate Get/Set Attribute function, such as :c:`${config['c_function_prefix']}SetAttributeViInt32()`.
```

Comment:
* I don't think we should talk about how this works. The fact that there is an "underlying driver function call" is an implementation detail to the Python customer. We should document this purely from the perspective of the Python customer.
* We should call them "properties" in Python, since they are presented as Python properties.

Reply:
I'm going to take a stab, feel free to improve or push back:

>This class supports repeated capabilites. These are multiple instances of the same type of functionality within the session.
>Repeated capability containers are properties of `Session` and can be indexed using a string, list, tuple, or slice (range).
> The repeated capability containers supported by this driver are:
```

The prefix is an implementation detail in my opinion. As a fact, I think we should downplay the fact that you can pass a string at all, instead use the more pythonic aproaches whenever we have examples.

What do you think?

Reply:
Removed references to and examples of prefix.
Updated overall description and per rep cap description. The per rep cap description seems redundant though.

![image](https://user-images.githubusercontent.com/11483903/67036618-8e774600-f0e1-11e9-80b0-acd9fed5a1da.png)